activities/supporting-codebase-governance/technical-roadmap-template.md

```diff
@@ -0,0 +1,58 @@
+---
+tags: Templates
+---
+
+# Technical Roadmap Template
+
+This is a template that can be used for technical roadmaps, it's based on a version that [Sharetribe used](https://github.com/sharetribe/sharetribe/blob/a029d02ab3b60009ebb6faff54117ba1034a6bfc/TECHNICAL_ROADMAP.md).
+
+## Overview
+
+This document sheds light on the current development plans of the team.
+Plans change constantly as new information comes to light.
+Thus, this document represents **a snapshot of our plans at this point in time**, based on the information that is currently available.
+
+In other words, the purpose of this document is to share the current state of the team's understanding.
+
+To reiterate: the plans shared in this document should not be considered set in stone.
+They will change when relevant information is absorbed by the team.
+
+## What is currently being worked on
+
+*Not all sections need to be filled in, but the title and why are recommended.*
+
+### Technical thing 1
+
+#### Why 1
+
+#### How 1
+
+#### What 1
+
+### Technical thing 2
+
+#### Why 2
+
+#### How 2
+
+#### What 2
+
+## Themes that will likely be worked on next
+
+These are the plans that currently feel like the best options for what the team should work on next.
+Improvements listed below might come out in a month, a year, or (unfortunately) never — it all depends what the team learns along the way.
+
+### Technical thing 3
```

Comment:
It can be clearer that this title needs to be replaced. I recommend the square bracket syntax seen on a lot of our other guides.

```suggestion
### [Technical thing name or title]
```